src/Managers/DataManager.py

```diff
@@ -16,17 +16,32 @@ def __init__(self, data: Data, model: Model) -> None:
         """Initialize the internal data object."""
         self._data = data
         self._model = model
+        """row_begin, row_end, columns begin, columns end"""
+        self._range = [0, 0, 0, 0]
+
 
     def upload_file(self, file_path: Any) -> None:
         """Read file to the pandas format and store it."""
         self._data.file = pd.read_csv(
             filepath_or_buffer=file_path, header=0, skipinitialspace=True
         )
         self._data.columns = list(self._data.file.columns)
+    def set_range_grid(self, begin_r: Protocol, end_r: Protocol,
+                       begin_c: Protocol, end_c: Protocol) -> None:
+        self._range[0] = begin_r
+        self._range[1] = end_r
+        self._range[2] = begin_c
+        self._range[3] = end_c
 
+        out_txt="Rows from {0} to {1} and Columns from {2} to {3}"
+        print(out_txt.format(self._range[0],
+                                 self._range[1],
+                                 self._range[2],
+                                 self._range[3]))
```

Comment:
I think we don't really need to print ranges here, because they will be shown on sliders anyway.